src/helpers/CRMApi.js

```diff
@@ -61,28 +61,27 @@ const getUsersSkills = async () => {
   });
 
   const usersSkillsReq = await axios.get(`${crmUrl}/ssw_userskills`);
-  const usersSkills = usersSkillsReq.data.value
-    .map((us) => {
-      const skill = skills.find((s) => s.ssw_skillid === us._ssw_skillid_value);
-      const marketingPage =
-        skill?.[
-          '_ssw_marketingpage_value@OData.Community.Display.V1.FormattedValue'
-        ] ?? '';
-
-      const userSkill = {
-        userId: us._ssw_systemuserid_value,
-        experienceLevel: getExperienceLevel(us.ssw_level),
-        sortOrder: us.ssw_sortorder || null,
-        technology: skill?.ssw_name ?? '',
-        marketingPage,
-        marketingPageUrl: consultingPages[marketingPage],
-        published:
-          skill?.['statuscode@OData.Community.Display.V1.FormattedValue'] ?? '',
-      };
-
-      return userSkill;
-    })
-    .filter((us) => us.published === 'Published');
+  const usersSkills = usersSkillsReq.data.value.map((us) => {
```

Comment:
Looks good! 

1. Does clicking on a skill if there is no marketing page try to take you to the page?

Reply:
No it doesn't. It would be only an `a` tag without an associated href

Reply:
I think it would make more sense if there wasn't a link to a marketing page that it doesn't underline the skill and isn't clickable. 

Is that an easy change?

Reply:
yes it's a good idea!
Please see my new commit, it's now using `span` instead of a link when it comes to no marketing url

![image](https://github.com/SSWConsulting/SSW.People/assets/127192800/7debf2b2-f0c2-4b97-b2b6-16012f2a3c2d)